recipe/meta.yaml

```diff
@@ -1,8 +1,8 @@
 {% set name = "analysis" %}
-{% set year = "2019" %}
-{% set cycle = "3" %}
-{% set version = "1" %}
-{% set micro = "1rc3" %}
+{% set year = "2020" %}
+{% set cycle = "1" %}
+{% set version = "0" %}
+{% set micro = "0rc0" %}
```

Comment:
I personally don't like this long version. Any objections I remove the extra `0`? It would look like `analysis-2020C1.0r0`.

@tacaswell, @danielballan, @stuartcampbell, any objections?

Reply:
>  It would look like `analysis-2020C1.0r0`.

You are proposing removing the `c` from `rc` as well?

Reply:
No, that was a typo. Will correct it.

Reply:
That is fine with me.

Reply:
Done, corrected the title of this PR as well.